app-android-xamarin/Moodis/Moodis/Extensions/IListExtensions.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Moodis.Extensions
+{
+    public static class IListExtensions
+    {
+        public static bool IsNullOrEmpty<T>(this IList<T> list)
+        {
+            if(list == null || list.Count == 0)
```

Comment:
I think you could write return list == null || list.Count == 0 instead this whole block

Reply:
Finally you started to review PRs !!! 🔥💯🔥